requirements/dev.txt

```diff
@@ -1,4 +1,4 @@
-# SHA1:0e3b4eb4275a8a7777e75467fd7ca031a2d61a9c
+# SHA1:81cd2388f255663e0c0693dada64e57d36d756f3
```

Comment:
I guess these are best re-generated once the pip-compile-multi PR lands first?

Reply:
Yeah I would re-run it